src/engine/bank06.asm

```diff
@@ -147,125 +147,159 @@ Func_18086: ; 18086 (6:4086)
 	ret
 ; 0x180d5
 
-Func_180d5: ; 180d5 (6:40d5)
+; this function is called when the player
```

Comment:
You can probably expand these lines a bit, 80-ish chars should be fine. To be honest, I never really bothered much about this myself and instead just went which however looked better (so it proabably isn't very consistent across)